distribution/lib/Standard/Base/0.0.0-dev/src/Data.enso

```diff
@@ -213,6 +206,35 @@ fetch (uri:(URI | Text)) (method:HTTP_Method=HTTP_Method.Get) (headers:(Vector (
      - File: shorthand for `Request_Body.Binary that_file`.
      - Any other Enso object: shorthand for `Request_Body.Json that_object`.
 
+   ! Specifying Content Types
+
+     If the `body` parameter specifies an explicit content type, then it is an
+     error to also specify additional `Content-Type` headers in the `headers`
+     parameter. (It is not an error to specify multiple `Content-Type` values in
+     `headers`, however.)
+
+   ! Default Content Types
+
+     The following specifies the default content type for each `Request_Body`
+     type.
+
+     - Request_Body.Text: `text/plain`
+     - Request_Body.Json: `application/json`
+     - Request_Body.Binary: `application/octet-stream`
+     - Request_Body.Form_Data:
+           If `url_encoded` is True: `application/x-www-form-urlencoded`
+           If `url_encoded` is False: `multipart/form-data`
+     - Request_Body.Empty: No content type is sent
+     - Text: `text/plain`
+     - File: `application/octet-stream`
+     - Any other Enso object: `application/json`
+
+   ! Specifying Content Encodings
+
+     The content encoding specified in the `Request_Body` is used to encode the
+     body. The `headers` list can contain additional `Content-Encoding` headers,
+     and these are sent to the server, but not used for encoding.
```

Comment:
👍

Reply:
Isn't Content-Encoding for compression? Are we supporting gzip deflate etc?
Its passed as a `charset=...` on the content type.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding